docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       - mailhog
     restart: always
     command: php -t /srv -S 0.0.0.0:80 .ht.router.php
+#    command: php -d zend_extension=xdebug.so -t /srv -S 0.0.0.0:80 .ht.router.php
```

Comment:
This line is needed because we bundle XDebug now, so here should be a quick way to use it